main.go

```diff
@@ -27,8 +30,9 @@ func main() {
 	viper.AddConfigPath("$HOME")
 	viper.SetConfigName(".spotify-cli")
 	viper.SetConfigType("json")
-	viper.SafeWriteConfig()
-	viper.ReadInConfig()
+	_ = viper.SafeWriteConfig()
+	err := viper.ReadInConfig()
+	cobra.CheckErr(errors.Wrap(err, "Failed to re-read configuration file."))
```

Comment:
The reason for not checking this error is that a malformed/unreadable config prompts the user to login, and then the config gets recreated on login. Showing an error here would require the user to hunt down the config file and delete/fix it themselves, which is definitely bad UX.
```suggestion
	_ = viper.ReadInConfig()
```